spec/Book/book_spec.rb

```diff
@@ -0,0 +1,33 @@
+require_relative '../spec_helper'
+
```

Comment:
- [ ] As required for the project to have unit tests to all public methods in all classes should, kindly ensure that you also implement tests for the `Item` class and to the public method inside.